client/cypress/e2e/settings.cy.ts

```diff
@@ -184,5 +188,37 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight }) =>
 
       cy.get('[data-test=EarnView__TipTile--connectWallet]').should('not.exist');
     });
+
+    it('Should show correct setting links', () => {
```

Comment:
All test cases start with a lowercase letter, please adjust. 
```suggestion
    it('should show correct settings links', () => {
```

Reply:
ok